### PR TITLE
Adding accessibility props and exposing animation parameters as props

### PR DESCRIPTION
**Pull request by luk3skyw4lker**
Exposed animation duration and easing as props as asked in #14 and added some accessibility props as asked in #13

### CODE REVIEW

**Comment by poberwong** [comment]
Looks good!
Will be reviewed tonight.

**Comment by luk3skyw4lker** [comment]
Thanks!

File: Example/App.js [hunk 1]
```diff
@@ -8,16 +8,20 @@ import {
...
-  state={
-    value: false,
-    value1: true
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: false,
+      value1: true
+    }
```
Comment (poberwong) [inline Example/App.js:17]: it may be unnecessary.

**Comment by poberwong** [comment]
Considering about many "pull request", I determinate to develop 2.0 to upgrade to [react-native@latest](https://github.com/facebook/react-native/releases/latest).

Thanks for your contribution.

**Pull request closed without merging**